### PR TITLE
don't use global loggers

### PR DESCRIPTION
**Pull request by egonelbre**
What: multiple places were using zap.L(), pass in the properly scoped logger.

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: pkg/certificates/certificates_test.go [hunk 1]
```diff
@@ -609,7 +609,7 @@ func TestCertificateSigner_Sign_E2E(t *testing.T) {
 				require.NoError(t, err)
 				require.NotNil(t, serverOpts)
 
-				service, err := server.New(serverOpts, sc.Address, sc.PrivateAddress, nil, config)
+				service, err := server.New(zap.NewNop(), serverOpts, sc.Address, sc.PrivateAddress, nil, config)
```
Comment (stefanbenten) [inline pkg/certificates/certificates_test.go]: Why do we have the NewNop() Logger here?

Comment (egonelbre) [reply pkg/certificates/certificates_test.go]: fixed

File: pkg/server/interceptors.go [hunk 2]
```diff
@@ -29,20 +29,19 @@ func logOnErrorStreamInterceptor(srv interface{}, ss grpc.ServerStream, info *gr
 			err == io.EOF {
 			return err
 		}
-		zap.S().Errorf("%+v", err)
+		server.log.Sugar().Errorf("%+v", err)
```
Comment (ifraixedes) [inline pkg/server/interceptors.go]: Not to change in this branch, but should we use the normal log rather than the sugared one?

Comment (egonelbre) [reply pkg/server/interceptors.go]: Tried to avoid figuring this out, but probably yes.

Comment (ifraixedes) [reply pkg/server/interceptors.go]: It was a chance to ask, I saw it on Monday while working on something and it popped up here.

**Pull request merged**